test/redis/ick_test.rb

```diff
@@ -1169,5 +1169,47 @@ def test_redis_is_available
         end
       end
     end
+
+    def test_big_test
```

Comment:
This tests defaults to fast and quiet, but I developed these optimizations running a perf test like so:
```
env REDIS_URL=redis://localhost:7379 BIG_TEST_VERBOSE=true BIG_TEST_CHUNK_SIZE=100000 BIG_TEST_NUM_CHUNKS=4 bundle _1.16.6_ exec rake test
```